### DISCUSSION TITLE
花火学园 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#61] [模板讨论]
花火学园 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by wsdjj** [comment]
大佬，我把flaresolverr 服务地址填上去显示405
![搜狗截图20230115170246](https://user-images.githubusercontent.com/76938752/212532213-b6a691ff-7fc4-4dc6-8243-e21ab05f1ed8.png)
请问flaresolverr是这样子直接在docker安装吗
![搜狗截图20230115170407](https://user-images.githubusercontent.com/76938752/212532322-7ad04c39-df00-478e-bb17-26097d450fd0.png)

**Reply by wjf0214** [reply]
应该是地址错了，`http://localhost:8191/v1`后面的v1也要。而且要容器内能访问到的。
以及目前官方的`flaresolverr`容器服务目前有点问题。cookie传递失败，我暂时用的别人修改的`15cm/FlareSolverr`版本，等官方更新合并了。

**Comment by wsdjj** [comment]
大佬我也改成15cm/FlareSolverr版本了
![搜狗截图20230116192945](https://user-images.githubusercontent.com/76938752/212668331-6ef8e133-c140-4267-88e5-3dbed5186820.png)
现在试了一下第一个到第三个包都正常的
![搜狗截图20230116193106](https://user-images.githubusercontent.com/76938752/212668562-2ae3ad74-32f4-42f0-bd55-193be6f98ef1.png)
就是到第四个包（签到页获取cf参数）就不行了
![搜狗截图20230116193249](https://user-images.githubusercontent.com/76938752/212668766-defe90b9-4183-430f-b13b-448fdc818510.png)
显示Unable to solve captcha of type turnstile不知道是为什么

**Reply by wjf0214** [reply]
看提示是`无法解决旋转门类型的验证码`的意思。可能网站还有其他验证？
然后 FlareSolverr 过不了？
我测试做模板的时候就签到页会出现cf的5秒盾。没遇到这种情况。我是返回的`Challenge solved!`
以及前面的登录，首页都是正常的吗？
我没法复现这种情况。

**Comment by wsdjj** [comment]
登录，首页都没有问题
![搜狗截图20230118160817](https://user-images.githubusercontent.com/76938752/213117427-50b0114e-cf84-4afb-900e-004d3a1139e6.png)
![搜狗截图20230118160837](https://user-images.githubusercontent.com/76938752/213117433-d48600af-3c18-4466-86c1-4ec142f1aef0.png)
就是第四个包（签到页获取cf参数）显示Unable to solve captcha of type turnstile，然后第五个包显示403了
![搜狗截图20230118161106](https://user-images.githubusercontent.com/76938752/213117800-d60cbfd5-a084-4160-8a2a-bfbd32a7de14.png)

**Comment by wsdjj** [comment]
大佬可以用了，我怀疑是我的网络问题(；′⌒`)
![搜狗截图20230120133938](https://user-images.githubusercontent.com/76938752/213625444-821eac22-52a6-4f2f-b624-383592bf43ac.png)

**Reply by wjf0214** [reply]
OK，可以用就行。

**Comment by VwEl** [comment]
有啥解决方案嘛
Failed at 4/8 request,
Fail assert: {"re": "\"status\": \"error\"", "from": "content"} from failed_asserts,
Request URL: {{flaresolverrUrl}}


{"status": "error", "message": "Unable to solve captcha of type turnstile", "solution": null, "startTimestamp": 1676475269053, "endTimestamp": 1676475279809, "version": "3.0.0"}

**Reply by wjf0214** [reply]
这个只能多尝试了。flaresolverr这个东西这个版本好像是经常出现这种报错。目前没啥好办法。官方3.1.0版本到现在也没出来。

**Comment by LeticiaAnna** [comment]
Failed at 1/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Unknown,
Request URL: {{flaresolverrUrl}}

flaresolverr 服务的地址 这个如何安装呀

**Reply by wjf0214** [reply]
安装[flaresolverr/flaresolverr](https://hub.docker.com/r/flaresolverr/flaresolverr)容器服务。
填写服务地址，类似`http://localhost:8191/v1`的地址。

**Comment by wsdjj** [comment]
大佬，flaresolverr一直显示超时
Failed at 1/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Connection timed out after 30001 milliseconds,
Request URL: {{flaresolverrUrl}}
flaresolverr镜像已经是最新的了
![搜狗截图20230705163225](https://github.com/wjf0214/qd-templates/assets/76938752/8db10eef-f215-4683-ad53-cd2e4700e6f9)
网络也是放一块的
![搜狗截图20230705163518](https://github.com/wjf0214/qd-templates/assets/76938752/c0adfb91-dffe-46aa-b0b3-acd0979ae52e)
不知道为什么好像一直连接不上

**Reply by wjf0214** [reply]
地址是不是少填了v1，是`http://xxx.xxxxx.com/v1`格式

**Reply by wsdjj** [reply]
没有，地址应该没有填错
![搜狗截图20230707160647](https://github.com/wjf0214/qd-templates/assets/76938752/95fce2f6-e0f9-4e21-8d34-854a0042fe22)

**Reply by wsdjj** [reply]
不知道为什么一直都是超时
![搜狗截图20230707160853](https://github.com/wjf0214/qd-templates/assets/76938752/6987a9f0-ddb6-459e-9ae5-b8ac01d718e0)

**Reply by wjf0214** [reply]
我偶尔也会超时。可以后续观察看看。或者调大点超时时间。

**Comment by a15355447898a** [comment]
我遇到了这个错误
```
Failed at 3/7 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 405: Method Not Allowed,
Request URL: https://www.sayhuahuo.xyz/member.php?mod=logging&action=login&loginsubmit=yes&loginhash={{loginhash|urlencode}}&inajax=1
```
